test/scenarios.py

```diff
@@ -9874,4 +9758,802 @@
             ),
         },
     ),
+    Scenario(
+        "scenario-analytics-1",
```

Comment:
I think it's better to name them the same, but at the same using tags

Reply:
good point, forgot about the tags, added them